biomejs/biome
# docs(website): spanish translation #1380

## Summary

Translate the following routes.

- [x] / (new home page)
- [x] sidebar
- [x] analyzer/index

## Test Plan

- Check if there is no missing translation in the translated files.
- Check if the translations are correct and if they adhere to the grammatical rules of Spanish.
